thrift/shared.thrift

```diff
@@ -1843,12 +1843,13 @@ struct CrossClusterTaskRequest {
 struct CrossClusterTaskResponse {
   10: optional i64 (js.type = "Long") taskID
   20: optional CrossClusterTaskType taskType
-  30: optional CrossClusterTaskFailedCause failedCause
-  40: optional CrossClusterStartChildExecutionResponseAttributes startChildExecutionAttributes
-  50: optional CrossClusterCancelExecutionResponseAttributes cancelExecutionAttributes
-  60: optional CrossClusterSignalExecutionResponseAttributes signalExecutionAttributes
-  70: optional CrossClusterRecordChildWorkflowExecutionCompleteResponseAttributes recordChildWorkflowExecutionCompleteAttributes
-  80: optional CrossClusterApplyParentClosePolicyResponseAttributes applyParentClosePolicyAttributes
+  30: optional i16 taskState
```

Comment:
it's ok to make breaking changes now since the feature is still under development.